config/defaults.go

```diff
@@ -9,6 +9,7 @@ import (
 const (
 	// DefaultListenAddress is a default listen address for P2P client.
 	DefaultListenAddress = "/ip4/0.0.0.0/tcp/7676"
+	Version              = "0.4.0"
```

Comment:
I think it's OK for now, but we need to think about injecting version information during build. 

I created issue for this: https://github.com/celestiaorg/cosmos-sdk-rollmint/issues/6.

Reply:
yes, i was thinking the same. if the version comes from the upstream config, we can easily utilize it, instead of having the manage the two versions. thanks for creating the issue.